src/test/java/com/casadocodigo/site/cadastroautor/NovoAutorEmailUnicoValidatorTest.java

```diff
@@ -0,0 +1,53 @@
+package com.casadocodigo.site.cadastroautor;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.validation.BeanPropertyBindingResult;
+import org.springframework.validation.Errors;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class NovoAutorEmailUnicoValidatorTest {
+
+    @Test
+    @DisplayName("Deve adicionar um erro ao objeto Errors do spring quando o email do autor ja esta cadastrado")
+    void deveAdicionarUmErroAoObjetoErrorsDoSpringQuandoOEmailDoAutorJaEstaCadastrado() {
+        String emailExistente = "henrique@email.com";
+
+        NovoAutorForm novoAutorForm = new NovoAutorForm("Jose", emailExistente, "descricao legal");
+
+        AutorRepository autorRepository = Mockito.mock(AutorRepository.class);
+        NovoAutorEmailUnicoValidator novoAutorEmailUnicoValidator = new NovoAutorEmailUnicoValidator(autorRepository);
```

Comment:
É comum a gente fazer import estático do Mockito. Aí ficaria:

```suggestion
        AutorRepository autorRepository = mock(AutorRepository.class);
        NovoAutorEmailUnicoValidator novoAutorEmailUnicoValidator = new NovoAutorEmailUnicoValidator(autorRepository);
```

Reply:
Entendo, foi mais por costume mesmo 😬